MainSamplesFall18.txt

```diff
@@ -1,277 +1,342 @@
-WGToLNuG_01J_5f_TuneCP5_13TeV-amcatnloFXFX-pythia8	0	27000000	RunIIFall18wmLHEGS	SMP
-WGToLNuG_01J_5f_TuneCP5_13TeV-amcatnloFXFX-pythia8	1	27000000	RunIIFall18wmLHEGS	SMP
-WJetsToLNu_0J_TuneCP5_13TeV-amcatnloFXFX-pythia8	0	200000000	RunIIFall18wmLHEGS	SMP
-WJetsToLNu_1J_TuneCP5_13TeV-amcatnloFXFX-pythia8	0	200000000	RunIIFall18wmLHEGS	SMP
-WJetsToLNu_2J_TuneCP5_13TeV-amcatnloFXFX-pythia8	0	100000000	RunIIFall18wmLHEGS	SMP
-WJetsToLNu_BGenFilter_Wpt-100to200_TuneCP5_13TeV-madgraphMLM-pythia8	0	31000000	RunIIFall18wmLHEGS	HIG
-WJetsToLNu_HT-100To200_TuneCP5_13TeV-madgraphMLM-pythia8	0	30000000	RunIIFall18wmLHEGS	SUS
-WJetsToLNu_HT-200To400_TuneCP5_13TeV-madgraphMLM-pythia8	0	20000000	RunIIFall18wmLHEGS	SUS
-WJetsToLNu_HT-600To800_TuneCP5_13TeV-madgraphMLM-pythia8	0	20000000	RunIIFall18wmLHEGS	SUS
-WJetsToLNu_HT-70To100_TuneCP5_13TeV-madgraphMLM-pythia8	0	20000000	RunIIFall18wmLHEGS	SUS
-WJetsToLNu_Pt-100To250_TuneCP5_13TeV-amcatnloFXFX-pythia8	0	100000000	RunIIFall18wmLHEGS	SMP
-WJetsToLNu_Pt-50To100_TuneCP5_13TeV-amcatnloFXFX-pythia8	0	20000000	RunIIFall18wmLHEGS	SMP
-WJetsToLNu_TuneCP5_13TeV-madgraphMLM-pythia8	0	80000000	RunIIFall18wmLHEGS	HIG
-WJetsToLNu_TuneCP5_13TeV-madgraphMLM-pythia8	0	80000000	RunIIFall18wmLHEGS	HIG
-WJetsToLNu_TuneCP5_13TeV-madgraphMLM-pythia8	1	80000000	RunIIFall18wmLHEGS	TAU
-WWToLNuQQ_NNPDF31_TuneCP5_13TeV-powheg-pythia8	0	20000000	RunIIFall18wmLHEGS	HIG
-WZTo1L1Nu2Q_13TeV_amcatnloFXFX_madspin_pythia8	0	20000000	RunIIFall18wmLHEGS	EXO
-WZTo2L2Q_13TeV_amcatnloFXFX_madspin_pythia8	0	30000000	RunIIFall18wmLHEGS	HIG
-WZTo3LNu_mllmin01_NNPDF31_TuneCP5_13TeV_powheg_pythia8	0	90000000	RunIIFall18wmLHEGS	HIG
-ZBJetsToNuNu_Zpt-100to200_TuneCP5_13TeV-madgraphMLM-pythia8	0	55000000	RunIIFall18wmLHEGS	HIG
-ZGToLLG_01J_5f_lowMLL_lowGPt_TuneCP5_13TeV-amcatnloFXFX-pythia8	0	80000000	RunIIFall18wmLHEGS	EXO
-ZGToLLG_01J_5f_lowMLL_lowGPt_TuneCP5_13TeV-amcatnloFXFX-pythia8	0	80000000	RunIIFall18wmLHEGS	EXO
-ZGToLLG_01J_LoosePtlPtg_5f_TuneCP5_13TeV-amcatnloFXFX-pythia8	0	20000000	RunIIFall18wmLHEGS	SMP
-ZH_HToWW_HpT150_WToHad_ZToLL_M125_TuneCP5_13TeV_powheg_pythia8	0	50000000	RunIIFall18wmLHEGS	HIG
-ZJetsToNuNu_HT-100To200_13TeV-madgraph	0	25000000	RunIIFall18wmLHEGS	SUS
-ZJetsToNuNu_HT-200To400_13TeV-madgraph	0	25000000	RunIIFall18wmLHEGS	SUS
-ZZTo2L2Nu_TuneCP5_13TeV_powheg_pythia8	2	50000000	RunIIFall18wmLHEGS	HIG
-ZZTo2L2Q_13TeV_amcatnloFXFX_madspin_pythia8	0	30000000	RunIIFall18wmLHEGS	HIG
-ZZTo2Q2Nu_TuneCP5_13TeV_amcatnloFXFX_madspin_pythia8	0	60000000	RunIIFall18wmLHEGS	SMP
-ZZTo4L_13TeV_powheg_pythia8_TuneCP5	0	20000000	RunIIFall18wmLHEGS	HIG
-ZZTo4L_M-1toInf_TuneCP5_13TeV_powheg_pythia8	0	72500000	RunIIFall18wmLHEGS	SUS
-ZZTo4L_TuneCP5_13TeV-amcatnloFXFX-pythia8	0	27000000	RunIIFall18wmLHEGS	SMP
-ZZTo4L_TuneCP5_13TeV_powheg_pythia8	2	100000000	RunIIFall18wmLHEGS	HIG
\ No newline at end of file
+TT_Mtt-1000toInf_TuneCP5_PSweights_13TeV-powheg-pythia8 0 24000000 RunIIFall18wmLHEGS B2G B2G-RunIIFall18wmLHEGS-00046 
```

Comment:
I see that this file is generated using update_twiki.py, so there is no point in storing it in git repository.